packages/frontend/apps/electron/src/main/application-menu/subject.ts

```
@@ -1,7 +1,9 @@
 import { Subject } from 'rxjs';
 
+export type NewPageAction = 'page' | 'edgeless' | 'default';
+
 export const applicationMenuSubjects = {
-  newPageAction$: new Subject<'page' | 'edgeless'>(),
+  newPageAction$: new Subject<NewPageAction>(),
```

ESLint

[error] Finnish notation should not be used here.

(rxjs/finnish)